### ISSUE TITLE
Improve Statistics Update OCR Checks And Accuracy

### ISSUE THREAD

**Issue by ormagoden** [becurrie/titandash#539]
**Describe the bug**
Highest Stage Reached detected wrong when update stats

**To Reproduce**
1. Stats settings -> Enable stats checked
2. Update Stats On Session Start

**Log Output Prior To Crash (Last 50 lines or so):**
```
[2020-04-27 18:19:14,302] INFO [TITANDASH INSTANCE 1] [BOT.PY:1311 - UPDATE_STATS() ] FORCING IN GAME STATISTICS UPDATE NOW.
[2020-04-27 18:19:31,059] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: HIGHEST_STAGE_REACHED -> 217
[2020-04-27 18:19:31,296] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: TOTAL_PET_LEVEL -> 4314
[2020-04-27 18:19:31,531] WARNING [TITANDASH INSTANCE 1] [STATS.PY:275 - UPDATE_OCR() ] NO DIGITS FOUND IN OCR RESULT, SKIPPING KEY: GOLD_EARNED
[2020-04-27 18:19:31,743] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: TAPS -> 8.49M
[2020-04-27 18:19:31,993] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: TITANS_KILLED -> 1.23M
[2020-04-27 18:19:32,229] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: BOSSES_KILLED -> 1.54M
[2020-04-27 18:19:32,458] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: CRITICAL_HITS -> 212M
[2020-04-27 18:19:32,707] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: CHESTERSONS_KILLED -> 492.03K
[2020-04-27 18:19:32,929] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: PRESTIGES -> 1559
[2020-04-27 18:19:33,203] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: DAYS_SINCE_INSTALL -> 1213
[2020-04-27 18:19:33,437] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: PLAY_TIME -> 48D 17:17:25
[2020-04-27 18:19:33,678] WARNING [TITANDASH INSTANCE 1] [STATS.PY:275 - UPDATE_OCR() ] NO DIGITS FOUND IN OCR RESULT, SKIPPING KEY: RELICS_EARNED
[2020-04-27 18:19:33,911] INFO [TITANDASH INSTANCE 1] [STATS.PY:315 - UPDATE_OCR() ] PARSED VALUE: FAIRIES_TAPPED -> 59.70K
[2020-04-27 18:19:34,143] WARNING [TITANDASH INSTANCE 1] [STATS.PY:275 - UPDATE_OCR() ] NO DIGITS FOUND IN OCR RESULT, SKIPPING KEY: DAILY_ACHIEVEMENTS
[2020-04-27 18:19:34,180] INFO [TITANDASH INSTANCE 1] [BOT.PY:485 - _CALCULATE() ] NEXT_STATS_UPDATE SHOULD BE EXECUTED IN 0D 01:00:00
```

**Additional context**
Here's game screen
![ms](https://user-images.githubusercontent.com/12068998/80367482-639e5380-88b5-11ea-84e1-7c558977cb9c.PNG)

**Comment by becurrie** [comment]
Thanks, I'll look into improving the quality and accuracy of the OCR here, this function is really old and should probably be updated anyways 👍

**Comment by becurrie** [comment]
Resolved by #541
